paper-fab-speed-dial.html

```diff
@@ -43,8 +45,8 @@
 			}
 
 			.close {
-				--paper-fab-background: var(--paper-grey-500);
-				--paper-fab-keyboard-focus-background: var(--paper-grey-500);
+				--paper-fab-background: var(--paper-fab-speed-dial-background-close);
```

Comment:
Thanks for the contribution. Could you please set the default to `--paper-grey-500` to keep backwards compatibility?

See https://www.polymer-project.org/2.0/docs/devguide/custom-css-properties (section "Create custom properties")

> You can also include a default value in the var() function, to use in case the user doesn't set the custom property:
> `color: var(--my-toolbar-title-color, blue);`
> To include a default value that is a custom property, use this syntax:
> `color: var(--my-color, var(--my-default-color))`

Reply:
Done :+1: